### ISSUE TITLE
Switch from bremcl/rootcerts to x/crypto/x509roots

### ISSUE THREAD

**Issue by stapelberg** [gokrazy/tools#59]
We first need to add a Bundle() accessor to the x509roots/fallback package, then we can dump the certificates (in `internal/packer/cacerts.go`) like so:

```go
	b := new(bytes.Buffer)
	for _, cert := range fallback.Bundle() {
		fmt.Fprintf(b, "# %s\n# %x\n", cert.Subject.String(), sha256.Sum256(cert.Raw))
		pem.Encode(b, &pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw})
	}

	return string(b.Bytes()), nil
```

**Comment by dolmen** [comment]
@stapelberg I'm working on it!

**Comment by dolmen** [comment]
> [@stapelberg](https://github.com/stapelberg) I'm working on it!

Nope. The `Bundle()` accessor isn't available.

**Comment by stapelberg** [comment]
Yeah, I filed https://github.com/golang/go/issues/69898 to export it. I’ll ping Roland again to see where this stands.

**Comment by dolmen** [comment]
What about this alternative: just to download the certificates directly from the Mozilla source while building the image and store the file where Go expects it.
